### DISCUSSION TITLE
Question about ontology for sam2

### DISCUSSION THREAD

**Discussion by Jarradmorden** [roboflow/notebooks#335] [Q&A] [answered]
### Search before asking

- [X] I have searched the Roboflow Notebooks [issues](https://github.com/roboflow-ai/notebooks/issues) and found no similar bug report.

### Notebook name

Hi this is more of a question about the ontology can it work to be trained to segment the rear of a car by prompting it "rear of car" for example or is it limited to more simpler learning like a car on it's own but not specific, thanks very much 

### Bug

No bug just question

### Environment

Windows 11

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by LinasKo** [comment]
Moving this to a discussion as it's not a bug.

**Comment by LinasKo** [comment, marked as answer]
From my experience - yes, it ought to be able to do that, much in the same way as it can detect faces and not the whole person. Place a positive marker on the car trunk, and multiple negative markers on other car parts.

Note that it's focused heavily on specific previously seen objects and may learn to track the back of a _specific_ car.

Instead, I'd check out the dataset [used in this notebook](https://github.com/roboflow/notebooks/blob/main/notebooks/train-yolov8-instance-segmentation-on-custom-dataset.ipynb) and train a detection or segmentation model.

**Reply by Jarradmorden** [reply]
Oh wow already exists perfect !

**Reply by LinasKo** [reply]
It can be hard to find, but we have the wildest datasets on [Universe](https://universe.roboflow.com/) :laughing:
